ocaml/xapi/message_forwarding.ml

```diff
@@ -3361,6 +3361,23 @@ functor
           (host_uuid ~__context host)
           value ;
         Local.Host.set_uefi_certificates ~__context ~host ~value
+
+      let host_sched_gran_to_string = function
```

Comment:
Would it be possible to use regular variants instead of polymorphic variants? (This would require a type definition)

Reply:
The polymorphic variant comes from the autogenerated code for enum types, so we can't change that here. However, to- and of-string functions are normally in record_util.ml, so we should move this function there.

Reply:
Even though `sched-gran` won't be a field of an `host` record?
I'd be happy to do that because it solves my problem of mutualizing the to_string function.